.github/workflows/pr_release_artifacts.yml

```diff
@@ -0,0 +1,137 @@
+name: pr release artifacts
+
+on:
+  schedule:
+    - cron: '0 15 * * 4'
```

Comment:
I was expecting a `pull_request` trigger here, to match the workflow name.

Reply:
That would mean it would trigger on every PR. I think the vast majority of PRs won't need to build the artifacts, so I decided to go with a `workflow_dispatch` trigger instead.